flow-api/src/main/java/com/sequenceiq/flow/api/FlowEndpoint.java

```diff
@@ -93,7 +94,7 @@ public interface FlowEndpoint {
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = "Get flow logs by a list of flow ids - Input size max 50", produces = "application/json", notes = "Flow log operations",
             nickname = "getFlowLogsByFlowIds")
-    Page<FlowLogResponse> getFlowLogsByFlowIds(@NotNull @Size(max = 50, message = "Input size should not be over 50")
+    FlowLogResponses getFlowLogsByFlowIds(@NotNull @Size(max = 50, message = "Input size should not be over 50")
         @QueryParam("flowIds") List<String> flowIds,
```

Comment:
if you are returning just a list - there is no need to create a DTO for that. just returning List<FlowLogResponse> will work fine.

Reply:
Addressed this and the below comments. Removed the DTOs and directly returned a list